PF/Assignment1/anshitaarya/prg5.java

```diff
@@ -0,0 +1,54 @@
+package anshitaarya;
+
+
+import java.util.Scanner;
+public class prg5 {
+	int checkOrder(int input[])
+	{
+		int i,j,count=0,count1=0,count2=0;
+		
+		for(i=0;i<input.length-1;i++)
```

Comment:
ACTION: This can be done by just iterating once instead of iterating two times. You can just compare first and second element of array if first element is less than second element of array that means array is either ascending sorted or not sorted. then you can iterate on array and check weather it maintains ascending sort order by comparing two consecutive elements, If it doesn't maintain sort order directly return 0. same thing you can do it in else (descending order) case .